sdk/eventhub/event-hubs/samples/receiveEvents.ts

```diff
@@ -45,8 +47,13 @@ async function main() {
       resolve();
     }, 30000);
   });
+
+  console.log(`Exiting receiveEvents sample`);
 }
 
-main().catch((err) => {
-  console.log("Error occurred: ", err);
-});
+if (!process.env["RUNNING_IN_TESTS"]) {
```

Comment:
Why put the env var on this as a gate? When we eventually run the samples in CI, we can take advantage of the exit code since all of our tooling already understands it. That'll be easier than writing a unit-test shim to run it.

Reply:
You mention it another comment - it is quite ugly. Perhaps I should make a helper function to do this instead?

Reply:
I moved the env var behind a simpler function to make it less confusing. Let me know what you think.